MidasCivil_Adapter/AdapterActions/Execute.cs

```diff
@@ -319,6 +320,52 @@ public async Task<bool> RunCommand(Analyse command)
         }
 
         /***************************************************/
+        public async Task<bool> RunCommand(ImportFile command)
+        {
+            if (m_midasCivilVersion == "9.5.0.nx")
+            {
+                string endpoint = "";
+                string filePath = command.FilePath;
+
+                if (File.Exists(filePath))
+                {
+                    string extension = Path.GetExtension(filePath).ToLower();
+
+                    if (extension == ".mct" || extension == ".mgt" || extension == ".txt")
+                        endpoint = "doc/IMPORTMXT";
+
+                    else if (extension == ".json")
+                        endpoint = "doc/IMPORT";
```

Comment:
Can we just remove this - I tried creating a simple JSON file using the Midas schema and the import did nothing.